gamedata/alldefs_post.lua

```diff
@@ -78,6 +78,9 @@ end
 function UnitDef_Post(name, uDef)
 	local modOptions = Spring.GetModOptions()
 
+	local isScav = string.sub(name, -5, -1) == "_scav"
+	local basename = isScav and string.sub(name, 1, -6) or name
```

Comment:
isn't there some `uDef.customparams.scav` that also gives the base name?

Reply:
In fact, there are custom params for both: `isscavenger` and `fromunit`